pkg/services/object/server.go

```diff
@@ -102,13 +105,22 @@ type FSChain interface {
 	LocalNodeUnderMaintenance() bool
 }
 
-// Storage groups ops of the node's object storage required to serve NeoFS API
-// Object service.
+type sessions interface {
```

Comment:
do not see why it should be a separate private interface

Reply:
Yeah, not really needed, but we can't delay SDK much.